### ISSUE TITLE
function VarToString is missing

### ISSUE THREAD

**Issue by nkgm** [qpkorr/vim-bufkill#13]
Reproducible when enabling `g:Debug`.

**Comment by pavoljuhas** [comment]
@qpkorr  - this patch should take care of that.

```patch
From 79584a738c514eb97827d89b4928b42a50d50755 Mon Sep 17 00:00:00 2001
From: Pavol Juhas <pavol.juhas@gmail.com>
Date: Fri, 24 Feb 2017 11:02:21 -0500
Subject: Remove call to undefined function VarToString.

Use builtin string() for conversion to string instead.
Fixes #13.
---
 plugin/bufkill.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugin/bufkill.vim b/plugin/bufkill.vim
index ba24118..ac87e2d 100644
--- a/plugin/bufkill.vim
+++ b/plugin/bufkill.vim
@@ -100,7 +100,7 @@ function! s:Debug(level, ...) "{{{1
       endif                                                                        " (Debug)
     endif                                                                          " (Debug)
     " Now print the value itself                                                     (Debug)
-    let s = s . VarToString(DebugArg)                                              " (Debug)
+    let s = s . string(DebugArg)                                                   " (Debug)
     if i < a:0                                                                     " (Debug)
       let s = s . ', '                                                             " (Debug)
     endif                                                                          " (Debug)

base-commit: 63f992f41eab6bcf050e741508d9af2f65d5f081
-- 
2.11.1
```

**Comment by qpkorr** [comment]
Thanks - think I had my own function when I wrote this plugin as 'string' either didn't exist or didn't do what I wanted - but now it seems more than sufficient.  Added.
